cookbook/deployment/heroku.rst

```diff
@@ -305,20 +305,20 @@ This is also very useful to build assets on the production system, e.g. with Ass
 
     With the next deploy, Heroku compiles your app using the Node.js buildpack and
     your npm packages become installed. On the other hand, your ``composer.json`` is
-    now ignored. To compile your app with both buildpacks, Node.js *and* PHP, you can
-    use a special `multiple buildpack`_. To override buildpack auto-detection, you
+    now ignored. To compile your app with both buildpacks, Node.js *and* PHP, you need 
+    use both buildpacks. To override buildpack auto-detection, you
     need to explicitly set the buildpack URL:
```

Comment:
You should remove ``URL`` here, as you code snippet sets the buildpack by its alias, not by its URL